docs/modules/processor/pages/logging-api.adoc

```diff
@@ -0,0 +1,90 @@
+= Logging
+
+All warning and error messages in Asciidoctor.js are routed through a logger.
+Many of the messages also include context about the source location (file, dir, path, lineno), which can be useful for integrations and tooling.
+
+When using the API, you can feed your own logger to the `LoggerManager` to capture, route, or observe the messages.
+
+== Default Logger
+
+The default logger will output all warning and error messages to `stderr` (the standard error output stream).
+
+Depending on your JavaScript environment, `stderr` will be resolved to:
+
+* https://developer.mozilla.org/fr/docs/Web/API/Console/warn[`console.warn`] function in a browser environment
+* https://nodejs.org/api/process.html#process_process_stderr[`process.stderr`] property in a Node.js environment
```

Comment:
Nashorn is still supported and it will use the `print` function to output to `stdout`: https://www.n-k.de/riding-the-nashorn/#_print_function

https://github.com/opal/opal/blob/master/stdlib/nashorn/io.rb

I will add another bullet :+1: